main.py

```diff
@@ -1,7 +1,38 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre, )
+
+    for first, last in actors:
+        Actor.objects.create(first_name=first, last_name=last)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(
+        first_name="George", last_name="Klooney"
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu", last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
+
+    filtered_smith = Actor.objects.filter(
+        last_name="Smith"
+    ).order_by("first_name")
+    return filtered_smith
```

Comment:
You don't necessarily need to create the variable `filtered_smith` in my opinion, just return the query set without it.

Reply:
I agree, you could skip it. What do you think?

Reply:
Better to add an empty line before return statement

Reply:
Understood, I will remember it 🌺